implementations.md

```diff
@@ -13,6 +13,8 @@ If you know of any associated projects that are not listed here, please file a p
 
 * [hyperhq/runv][runv] - Hypervisor-based runtime for OCI
 * [clearcontainers/runtime][cc-runtime] - Hypervisor-based OCI runtime utilising [virtcontainers][virtcontainers] by Intel®.
+* [kata-containers/runtime][kata-runtime] - Hypervisor-based OCI runtime
+  combining technology from [clearcontainers/runtime][cc-runtime] and [hyperhq/runv][runv].
```

Comment:
The line-wrap here doesn't seem quite right -- I think this should all be one line per https://github.com/opencontainers/runtime-spec/blob/master/style.md#one-sentence-per-line